[2025-08-09T20:14:15.108+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-09T20:14:15.136+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: xrs_update_12h.update_swpc_recent manual__2025-08-09T19:53:05+00:00 [queued]>
[2025-08-09T20:14:15.143+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: xrs_update_12h.update_swpc_recent manual__2025-08-09T19:53:05+00:00 [queued]>
[2025-08-09T20:14:15.143+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 4
[2025-08-09T20:14:15.153+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): update_swpc_recent> on 2025-08-09 19:53:05+00:00
[2025-08-09T20:14:15.161+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'xrs_update_12h', 'update_swpc_recent', 'manual__2025-08-09T19:53:05+00:00', '--job-id', '77', '--raw', '--subdir', 'DAGS_FOLDER/xrs_update_daily.py', '--cfg-path', '/tmp/tmpw_3p275i']
[2025-08-09T20:14:15.163+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=226) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-08-09T20:14:15.163+0000] {standard_task_runner.py:91} INFO - Job 77: Subtask update_swpc_recent
[2025-08-09T20:14:15.163+0000] {standard_task_runner.py:63} INFO - Started process 227 to run task
[2025-08-09T20:14:15.211+0000] {task_command.py:426} INFO - Running <TaskInstance: xrs_update_12h.update_swpc_recent manual__2025-08-09T19:53:05+00:00 [running]> on host 9d4e07229e62
[2025-08-09T20:14:15.289+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='you' AIRFLOW_CTX_DAG_ID='xrs_update_12h' AIRFLOW_CTX_TASK_ID='update_swpc_recent' AIRFLOW_CTX_EXECUTION_DATE='2025-08-09T19:53:05+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-09T19:53:05+00:00'
[2025-08-09T20:14:15.290+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-09T20:14:15.300+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-08-09T20:14:15.301+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/project/src/data/xrs_update.py --mode update --since-minutes 20160 --keep-days 180']
[2025-08-09T20:14:15.311+0000] {subprocess.py:86} INFO - Output:
[2025-08-09T20:14:16.862+0000] {subprocess.py:93} INFO - [INFO] retention applied: >= 2025-02-10 20:14:16.830518+00:00
[2025-08-09T20:14:16.862+0000] {subprocess.py:93} INFO - {"merged_rows_total": 10106, "range": ["2025-08-02 19:46:00+00:00", "2025-08-09 20:11:00+00:00"]}
[2025-08-09T20:14:16.964+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-08-09T20:14:16.965+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-09T20:14:16.989+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=xrs_update_12h, task_id=update_swpc_recent, run_id=manual__2025-08-09T19:53:05+00:00, execution_date=20250809T195305, start_date=20250809T201415, end_date=20250809T201416
[2025-08-09T20:14:17.021+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-08-09T20:14:17.040+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-08-09T20:14:17.044+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
